src/modules/recipes/recipeService.js

```diff
@@ -1,9 +1,14 @@
 
-export async function fetchRecipesByIngredient(idMeal) {
```

Comment:
I think you changed this function by mistake instead of just adding the `fetchRecipeDetailsById` below. No recipes were shown on the page as API request would not return any data. It's always a good thing to check what data you get in API response when you have issues like this.